tools/distpartitioning/utils.py

```diff
@@ -323,7 +376,8 @@ def write_dgl_objects(graph_obj, node_features, edge_features, output_dir, part_
 
     part_dir = output_dir + '/part' + str(part_id)
     os.makedirs(part_dir, exist_ok=True)
-    write_graph_dgl(os.path.join(part_dir ,'part'+str(part_id)), graph_obj)
+    #write_graph_dgl(os.path.join(part_dir ,'part'+str(part_id)), graph_obj)
```

Comment:
remove the commented code.

Reply:
removed the commented out line as per suggestion.